lib/WeBWorK/ContentGenerator/Instructor/PGProblemEditor.pm

```diff
@@ -599,7 +599,7 @@ sub backupFile ($c, $outputFilePath) {
 	}
 }
 
-sub saveFileChanges ($c, $outputFilePath, $backup = undef) {
+sub saveFileChanges ($c, $outputFilePath, $backup = 0) {
```

Comment:
Note that I misinterpreted the usage before when I said to make the default here `undef`.  I thought it was a string value, but it is a purely boolean value.  So `0` makes more sense for the default.